holoviews/plotting/bokeh/plot.py

```diff
@@ -355,6 +356,39 @@ def sync_sources(self):
         self.handles['shared_sources'] = shared_sources
         self.handles['source_cols'] = source_cols
 
+    def init_links(self):
```

Comment:
The three methods below are tightly coupled to each other but only loosely coupled to ``BokehPlot`` (only other use of ``self`` is ``self.traverse``). For this reason I would consider moving them to one of the appropriate ``Link`` classes.